android/src/test/java/com/adyenreactnativesdk/configuration/AnalyticsParserTest.kt

```diff
@@ -6,36 +6,52 @@ import org.junit.Test
 
 class AnalyticsParserTest {
 
+    @Test
+    fun applyConfigurationOnSubDictionary() {
```

Comment:
The other tests start with "test...", shall we keep this naming convention here as well?